nosnoc/model.py

```diff
@@ -32,6 +32,10 @@ class NosnocModel:
         different state equations with each boundary zone
     :param g_Stewart: List of stewart functions to define the regions (instead of S & c)
     :param u: controls
+    :param z: user defined algebraic state variables
+    :param z0: initial algebraic values
```

Comment:
I'd write `initial guess for user defined algebraic variables`.
I guess those could also be computed corresponding to `x0` with lambda00, but that is just a possible extension.

Reply:
Changed the comment. Agreed that this can be computed from `x0` (and optionally `alpha`) and `g_z`, but that can/will be added later.